trie/trieStorageManagerWithoutPruning.go

```diff
@@ -11,23 +11,23 @@ import (
 // trieStorageManagerWithoutPruning manages the storage operations of the trie, but does not prune old values
 type trieStorageManagerWithoutPruning struct {
 	common.StorageManager
-	storageManagerExtension
+	storage *trieStorageManager
```

Comment:
hard to test since now you glued the pointer here

Reply:
Reverted :grin: